src/elm/Page/Community/Settings/Features.elm

```diff
@@ -135,14 +136,27 @@ toggleView translations labelText isEnabled toggleFunction inputId =
 
             else
                 "text-grey"
+
+        kycTooltip =
+            if inputId == "kyc" then
+                span [ class "icon-tooltip inline-block align-center ml-1" ]
+                    [ Icons.question "inline-block"
+                    , div
+                        [ class "icon-tooltip-content" ]
+                        [ text (translate "community.kyc.info")
+                        ]
+                    ]
+
+            else
+                text ""
     in
     div
         [ class "grid w-full py-4"
         , style "grid-template" """
                                 'label status toggle' 40px / auto 80px 50px
                                 """
         ]
-        [ span [ classes, style "grid-area" "label" ] [ text labelText ]
+        [ span [ classes, style "grid-area" "label" ] [ text labelText, kycTooltip ]
```

Comment:
It may a little detail oriented but the icon is a little misaligned 
<img width="67" alt="Screen Shot 2020-09-28 at 11 05 30" src="https://user-images.githubusercontent.com/1082127/94442595-88576980-017a-11eb-90b9-c46217d2a219.png">

Reply:
Fixed! Also fixed the same issue on the Profile page and improved the code a bit.